realm/realm-library/src/main/java/io/realm/RealmCache.java

```diff
@@ -15,6 +15,8 @@
  */
 package io.realm;
 
+import android.text.TextUtils;
```

Comment:
we're already using `TextUtils` in the code base ex: https://github.com/realm/realm-java/blob/9cea9356930b4357403b46ef4ca9995598a078ce/realm/realm-library/src/main/java/io/realm/RealmConfiguration.java#L184